docs/configs.md

```diff
@@ -78,6 +78,7 @@ Name | Description | Default Value
 <a name="sql.incompatibleDateFormats.enabled"></a>spark.rapids.sql.incompatibleDateFormats.enabled|When parsing strings as dates and timestamps in functions like unix_timestamp, setting this to true will force all parsing onto GPU even for formats that can result in incorrect results when parsing invalid inputs.|false
 <a name="sql.incompatibleOps.enabled"></a>spark.rapids.sql.incompatibleOps.enabled|For operations that work, but are not 100% compatible with the Spark equivalent set if they should be enabled by default or disabled by default.|false
 <a name="sql.metrics.level"></a>spark.rapids.sql.metrics.level|GPU plans can produce a lot more metrics than CPU plans do. In very large queries this can sometimes result in going over the max result size limit for the driver. Supported values include DEBUG which will enable all metrics supported and typically only needs to be enabled when debugging the plugin. MODERATE which should output enough metrics to understand how long each part of the query is taking and how much data is going to each part of the query. ESSENTIAL which disables most metrics except those Apache Spark CPU plans will also report or their equivalents.|MODERATE
+<a name="sql.outOfCoreSort.enabled"></a>spark.rapids.sql.outOfCoreSort.enabled|enable or disable out of core sorting. Out of core sorting allows for sorting large skewed data sets, but is not a stable sort. Apache Spark does not guarantee a stable sort in most cases, except when coalescing all data to a single task and sorting there. In that case we will produce different results.|true
```

Comment:
I personally think that "out of core" or "external memory" would be the correct term to use. Spillable is very Spark specific, but that is the audience we are going for so if you insist I will change it. The only reason this is configurable is because of that one small use case where the stable sort matters. I have not seen enough of a performance difference to justify turning it off for performance. 

Perhaps we should just call it stable vs unstable sort, and in the future we can add in an out of core stable sort implementation.

Reply:
> if you insist I will change it.

I'm not insisting, just pointing out that the term will not be familiar to many Spark users reading the documentation and it's not really defined anywhere.